indico/MaKaC/services/implementation/conference.py

```diff
@@ -347,8 +348,7 @@ class ConferenceDefaultStyleModification(ConferenceTextModificationBase):
     Conference default style modification
     """
     def _handleSet(self):
-        styleMgr = HelperMaKaCInfo.getMaKaCInfoInstance().getStyleManager()
-        if self._value == styleMgr.getDefaultStyleForEventType(self._conf.getType()):
+        if self._value == theme_settings.defaults[self._conf.getType()]:
```

Comment:
what about using the `Event.type` property instead of `getType`? that would also let you use `lecture` instead of `simple_event`